libff/algebra/curves/alt_bn128/alt_bn128_pairing.cpp

```diff
@@ -238,56 +242,58 @@ alt_bn128_GT alt_bn128_final_exponentiation(const alt_bn128_Fq12 &elt)
+    const alt_bn128_Fq2 F = E + E + E;                   // F = 3 * E
+    const alt_bn128_Fq2 G = two_inv * (B + F);           // G = (B+F)/2
+    const alt_bn128_Fq2 H = (Y + Z).squared() - (B + C); // H = (Y1+Z1)^2-(B+C)
+    const alt_bn128_Fq2 I = E - B;                       // I = E-B
+    const alt_bn128_Fq2 J = X.squared();                 // J = X1^2
+    const alt_bn128_Fq2 E_squared = E.squared();         // E_squared = E^2
+
+    current.X = A * (B - F); // X3 = A * (B-F)
+    current.Y =
+        G.squared() - (E_squared + E_squared + E_squared); // Y3 = G^2 - 3*E^2
+    current.Z = B * H;                                     // Z3 = B * H
+    c.ell_0 = alt_bn128_twist * I;                         // ell_0 = xi * I
+    c.ell_VW = -H;        // ell_VW = - H (later: * yP)
+    c.ell_VV = J + J + J; // ell_VV = 3*J (later: * xP)
 }
 
-void mixed_addition_step_for_flipped_miller_loop(const alt_bn128_G2 base,
-                                                 alt_bn128_G2 &current,
-                                                 alt_bn128_ate_ell_coeffs &c)
+void mixed_addition_step_for_flipped_miller_loop(
+    const alt_bn128_G2 base, alt_bn128_G2 &current, alt_bn128_ate_ell_coeffs &c)
 {
     const alt_bn128_Fq2 X1 = current.X, Y1 = current.Y, Z1 = current.Z;
     const alt_bn128_Fq2 &x2 = base.X, &y2 = base.Y;
 
-    const alt_bn128_Fq2 D = X1 - x2 * Z1;          // D = X1 - X2*Z1
-    const alt_bn128_Fq2 E = Y1 - y2 * Z1;          // E = Y1 - Y2*Z1
-    const alt_bn128_Fq2 F = D.squared();           // F = D^2
-    const alt_bn128_Fq2 G = E.squared();           // G = E^2
-    const alt_bn128_Fq2 H = D*F;                   // H = D*F
-    const alt_bn128_Fq2 I = X1 * F;                // I = X1 * F
-    const alt_bn128_Fq2 J = H + Z1*G - (I+I);      // J = H + Z1*G - (I+I)
-
-    current.X = D * J;                           // X3 = D*J
-    current.Y = E * (I-J)-(H * Y1);              // Y3 = E*(I-J)-(H*Y1)
-    current.Z = Z1 * H;                          // Z3 = Z1*H
-    c.ell_0 = alt_bn128_twist * (E * x2 - D * y2); // ell_0 = xi * (E * X2 - D * Y2)
-    c.ell_VV = - E;                              // ell_VV = - E (later: * xP)
-    c.ell_VW = D;                                // ell_VW = D (later: * yP    )
+    const alt_bn128_Fq2 D = X1 - x2 * Z1;         // D = X1 - X2*Z1
+    const alt_bn128_Fq2 E = Y1 - y2 * Z1;         // E = Y1 - Y2*Z1
+    const alt_bn128_Fq2 F = D.squared();          // F = D^2
+    const alt_bn128_Fq2 G = E.squared();          // G = E^2
+    const alt_bn128_Fq2 H = D * F;                // H = D*F
+    const alt_bn128_Fq2 I = X1 * F;               // I = X1 * F
+    const alt_bn128_Fq2 J = H + Z1 * G - (I + I); // J = H + Z1*G - (I+I)
+
+    current.X = D * J;                  // X3 = D*J
+    current.Y = E * (I - J) - (H * Y1); // Y3 = E*(I-J)-(H*Y1)
+    current.Z = Z1 * H;                 // Z3 = Z1*H
+    c.ell_0 =
+        alt_bn128_twist * (E * x2 - D * y2); // ell_0 = xi * (E * X2 - D * Y2)
```

Comment:
Yes.. I definitely need to use some VIM magic to make sure these go away, these comments affect readability

Reply:
Move these to the previous line